transforms/message-remove-classic-theme/__testfixtures__/Basic.input.js

```diff
@@ -0,0 +1,11 @@
+import Message from "carbon-react/lib/components/message";
+export default () => <Message as="info" roundedCorners />;
+export const withProps = () => <Message as="info" title="Example" roundedCorners={true} />;
+const as = "info";
+const roundedCorners = true;
+export const asVariable = () => <Message as={as} roundedCorners={roundedCorners} />;
+const props = { as: "info", title: "Example", roundedCorners: true }
+export const spread = () => <Message {...props} roundedCorners />;
```

Comment:
I think you need to remove `roundedCorners` as an attribute here as it's getting spread out in props

Reply:
I intentionally added this test case to ensure that it removes all occurrences, not just the first instance.